src/rewriter.ts

```diff
@@ -51,8 +51,15 @@ export abstract class Rewriter {
   visit(node: ts.Node) {
     // this.logWithIndent('node: ' + ts.SyntaxKind[node.kind]);
     this.indent++;
-    if (!this.maybeProcess(node)) {
-      this.writeNode(node);
+    try {
+      if (!this.maybeProcess(node)) {
+        this.writeNode(node);
+      }
+    } catch (e) {
+      e.message += `\n at ${ts.SyntaxKind[node.kind]} in ${this.file.fileName}:`;
```

Comment:
Is it guaranteed that e will have a .message, even for stuff like divide by zero errors or whatever?  (I don't understand why exceptions in TS are type any)